static/bidder-info/taboola.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://{{.MediaType}}.bidder.taboola.com/OpenRTB/PS/auction/{{.GvlID}}/{{.PublisherID}}"
+endpoint: "https://{{.MediaType}}.bidder.taboola.com/OpenRTB/PS/auction?exchange={{.GvlID}}&publisher={{.PublisherID}}"
```

Comment:
Based on your description in the PR that this addresses the issue #2612, this endpoint is still using the dynamic host. You would need to use the static domain. For example, in this case, `MediaType` macro needs to be moved out as query param.

Reply:
@ahmadlob This is a fine change to make but this change definitely not addressing the issue #2612. For static host, macro should not be included in the endpoint domain.
The above host is still dynamic.
Please change the description of the PR to reflect the changes that you are doing.

Reply:
@gargcreation1992 changed the PR description and title